CloudinaryDotNet/Actions/AssetsUpload/UploadResult.cs

```diff
@@ -22,6 +22,20 @@ public abstract class UploadResult : BaseResult
         [DataMember(Name = "public_id")]
         public string PublicId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the folder where the asset is stored in the Media Library.
+        /// This value does not impact the asset’s Public ID.
```

Comment:
Unless use_asset_folder_as_public_id_prefix is set to true

Reply:
That information is written under that parameter description, no need to duplicate it.